1.1/spec/11-Part-08.adoc

```diff
@@ -492,40 +479,57 @@ The function http://www.opengis.net/def/function/geosparql/asKML[`geof:asKML`] c
 
-This section establishes the requirements for representing geometry data in RDF as represented in a Discrete Global Grid System (DGGS), in text. The form of representation is known as a _DGGS Well-Known Text_ geometry representation and is based on elements of the second version of the DGGS _Abstract Specification_ <<DGGSAS>>. It defines one RDFS Datatype:
-http://www.opengis.net/ont/geosparql#dggsLiteral[`+http://www.opengis.net/ont/geosparql#dggsLiteral+`] and one property, http://www.opengis.net/ont/geosparql#asDGGS[`+http://www.opengis.net/ont/geosparql#asDGGS+`].
+This section establishes the requirements for representing Discrete Global Grid System (DGGS) geometry data as RDF literals. The form of representation is specific to individual DGGS implementations: known DGGSes are not compatible or even very similar. 
+
+Here are defined two RDFS Datatypes:
+http://www.opengis.net/ont/geosparql#dggsLiteral[`+http://www.opengis.net/ont/geosparql#dggsLiteral+`] & http://www.opengis.net/ont/geosparql#auspixDggsLiteral[`+http://www.opengis.net/ont/geosparql#auspixDggsLiteral+`] and one property, http://www.opengis.net/ont/geosparql#asDGGS[`+http://www.opengis.net/ont/geosparql#asDGGS+`]. 
+
+NOTE: The two datatypes defined here are for an abstract DGGS implementation (http://www.opengis.net/ont/geosparql#dggsLiteral[`geo:dggsLiteral`]) and a specific one, AusPIX <<AUSPIX>> (http://www.opengis.net/ont/geosparql#auspixDggsLiteral[`geo:auspixDggsLiteral`]). The purposes of including both of these are to indicate the conceptual position of DGGS literals in this specification - the abstract form acts as a parent of all other potential, specific, DGGS datatypes - and to exemplify a specific system implementation which allows for real examples of use.
 
 ===== RDFS Datatype: geo:dggsLiteral
 
 ```turtle
 geo:dggsLiteral a rdfs:Datatype ;
     rdfs:isDefinedBy geo: ;
-    skos:prefLabel "DGGS Well-Known Text Literal"@en ;
-    skos:definition "A textual serialization of a Discrete Global Grid (DGGS) geometry object."@en .
+    skos:prefLabel "DGGS Literal"@en ;
+    skos:definition "A textual serialization of a Discrete Global Grid System (DGGS) geometry object."@en .
 ```
 
-Valid http://www.opengis.net/ont/geosparql#dggsLiteral[`geo:dggsLiteral`] instances are formed by encoding geometry information as text and as required by a particular DGGS and in accordance with the _Discrete Global Grid System Abstract Specification_ <<DGGSAS>>. An indication of the particular DGGS, as well as the geometric information must also be indicated in the literal as per the following _ABNF_ <<IETF5234>> syntax specification:
-
-```
-dggsLiteral ::= "<" IRI ">" LWSP geometric-data
-```
+Valid http://www.opengis.net/ont/geosparql#dggsLiteral[`geo:dggsLiteral`] instances are formed by encoding geometry information according to specific DGGS implmentation. The specific implementation should be indicated by use of a subclass of the `geo:dggsLiteral` datatype. 
```

Comment:
typo: implmentation

Reply:
Fixed as requested in commit https://github.com/opengeospatial/ogc-geosparql/pull/162/commits/e8fae58e396d5a1c9762cc55929e3ca0cfa17f9e